projection/validator_delegation/migrations/20220117111405_create_view_vd_redelegation_queue.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE view_vd_redelegation_queue (
+    id BIGSERIAL,
+    completion_time BIGINT NOT NULL UNIQUE,
+    dvv_triplets JSONB NOT NULL,
```

Comment:
Can we add some description for short formed column?

Reply:
Sure.